GameData/RP-1/Programs/Programs.cfg

```diff
@@ -287,11 +288,109 @@ RP0_PROGRAM
 		Fast = 600
 	}
 }
+RP0_PROGRAM
+{
+	name = EarlyEOS
+	title = Early Earth Observation Satellites
+	description = Now that you have proven the ability to send satellites into orbit, the next step is to send them into more useful orbits with beneficial payloads. The goal of this program is to put satellites with different payload requirements into specific and more challenging orbits. 
+	requirementsPrettyText = Complete Early Satellites program
+	objectivesPrettyText = Launch satellites to analyze weather, measure solar wind and cosmic rays, and determine the extent of the Magnetosphere.
+	nominalDurationYears = 3
+	baseFunding = 420000
+	repDeltaOnCompletePerYearEarly = 130
+	repPenaltyPerYearLate = 130
+	slots = 1
+
+	REQUIREMENTS
+	{
+		ANY
+		{
+			complete_program = EarlySatellites
+			complete_program = EarlySatellites-Heavy
+		}
+  		ANY
+    		{
+      			active_program = CrewedOrbit
```

Comment:
Uh, why does CrewedOrbit need to be active or completed?

Reply:
Because of the Corona contracts needing the first_OrbitReturn (or whatever the actual name is) completed.  I figured this way it would show up in the program screen as being needed, instead of being a surprise when someone accepts the contract and then find out they needed that contract done.

Reply:
I think it's a much better approach to make the first orbit return contract depend on either EOS1 or Crewed Early (in the same way you changed the commsat contracts), and make completing it required for both programs. You'll only have to fly it once of course, but this solves either doing Corona before that contract, or having EOS1 gated on Crewed Orbit.